### PR TITLE
Added links translated

### PR DESCRIPTION
**Pull request by FreakDroid**
Heya, I updated some links because I was busy. I hope the weekend I gonna update the rest.

Also I didn't know how translate some links because I don't know the correct word in Spanish.

Let me know anything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,11 +104,11 @@ Algunos recursos poseen un emoticón para ayudar a entender el tipo de contenido
 <link rel="icon" type="image/png" href="https://example.com/favicon.png">
 ```
 
-> * 🛠 [Favicon Generator](https://www.favicon-generator.org/)
-> * 🛠 [RealFaviconGenerator](https://realfavicongenerator.net/)
-> * 📖 [Favicon Cheat Sheet](https://github.com/audreyr/favicon-cheat-sheet)
-> * 📖 [Favicons, Touch Icons, Tile Icons, etc. Which Do You Need? - CSS Tricks](https://css-tricks.com/favicon-quiz/)
-> * 📖 [PNG favicons - caniuse](https://caniuse.com/#feat=link-icon-png)
+> * 🛠 [Generador de Favicon](https://www.favicon-generator.org/) `(Fuente en Inglés)`
```
Comment (eoasakura) [inline README.md:107]: Links that have the 🛠 icon translate as `(Herramienta en Inglés)`.

Comment (WilfredoRPL) [reply README.md:107]: So the other ones, like books and tools we will keep it in English??? or Will we change it later too?

Comment (eoasakura) [reply README.md:107]: For a complete translation its better to translate all titles, that way people that read this repo knows what links are about even if are in another language.
That way they can google the title to find information about the topic in Spanish if they can´t read English.

Thank you 😃

Comment (WilfredoRPL) [reply README.md:107]: So I gonna close this PR.

Thank you :smile:

**Pull request closed without merging**